README.md

```diff
@@ -13,7 +13,7 @@ URL to be log in to Moodle without typing username and password.
 
 Using
 -----
-1. Install the plugin as usual.
+1. Install the plugin in (Admin > Plugins > Install plugins).
```

Comment:
Someone like me never uses GUI for installing plugins :) 
Would be good in this case to have something like "Install the plugin using admin interface (Admin > Plugins > Install plugins) or command line"